src/UI/Component/Input/Container/Filter/ProxyFilterField.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/* Copyright (c) 2018 Thomas Famula <famula@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Input\Container\Filter;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+interface ProxyFilterField extends Component, JavaScriptBindable, Clickable {
```

Comment:
I did not wrap my head around this for long, why is this needed? Is this only for tagging elements? Is it to introduce the clickable behaviour to attach js? If so, maybe this is useful for more than just filters. 

Since I do not understand it imeediatly, leave a comment in the interface description.

Reply:
This is an internal class for the Filter Fields. It will not be offered through the factory. We can also remove it from the interfaces, if needed. This internal class wants to reuse framework concepts like JSBindable and Clickable, but I am pretty sure that this class is not useful for other components. It's very focused on the **Filter Field** internals. We will add some comments.

Reply:
IMO this could just go to the Implementation, then.